docs/tutorial/03-model-serving.ipynb

```diff
@@ -61,7 +61,8 @@
      "name": "stdout",
```

Comment:
<p>If you use <code>MLRUN_VERSION=1.3.0-rc22 make api</code> to build the server version the warning should disapper </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3079/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>